docker-compose.yml

```diff
@@ -3,6 +3,8 @@ services:
   db:
     image: postgres
     restart: on-failure
+    ports:
+      - '5432:5432'
```

Comment:
Why though? Are you wanting to access the db from outside the docker-compose images?